### PR TITLE
Iya models addition

### PR DESCRIPTION
**Pull request by Sumedh-ravi1604**
adding new calculations for IYA models, added new iya models in AppConstants

### CODE REVIEW

File: pgdjangosql/templates/src/utils/app-utils.js [hunk 1]
```diff
@@ -611,6 +611,30 @@ export function sortHeaderDate(headerDates, isOpSection = false) {
   return updatedHeaderDates;
 }
 
+// export function getAbsIyaVal(
```
Comment (nabendukarmakar) [inline pgdjangosql/templates/src/utils/app-utils.js]: please remove the code if not required.

File: pgdjangosql/templates/src/utils/app-utils.js [hunk 2]
```diff
@@ -624,10 +648,22 @@ export function getIyaKeyVal(
   if (flattenedKbdData) {
     let currKeyVal;
     let prevKeyVal;
-
+    let absIyaValues = [];
     const prevAbsKey = getPrevAbsKey(absIyaKey);
     const kbdName = getKbdName(kbd, measures.name);
-
+    // if (kbdName != 'BOP' && kbdName != 'Channel Wise Output' && kbdName != 'Output') {
```
Comment (nabendukarmakar) [inline pgdjangosql/templates/src/utils/app-utils.js]: please remove the code if not required

File: pgdjangosql/templates/src/utils/output-section-utils.js [hunk 3]
```diff
@@ -42,6 +43,53 @@ export function flattenKbdData(kbdData) {
...
+function getDataForIyaModels(modelData, measurename, absKey, absIyaCalulatedData, flattenedKbdData) {
+            let formulaActual = modelData.CHANNEL_CALCULATIONS[measurename];
+            let formulaPlanned = modelData.CHANNEL_CALCULATIONS[measurename];
+            let iyaValActual = [];
+            let iyaValPlanned = [];
+            let absPredictionActual;
+            let absPredictionPlanned;
+            modelData.KBD_LIST.forEach(kbdName => {
+            absIyaCalulatedData.map(iyaValue => {
+                  if (iyaValue.currKey === kbdName + ' > '+ measurename + ' > ' + 'Actual' + ' > ' + absKey){
```
Comment (nabendukarmakar) [inline pgdjangosql/templates/src/utils/output-section-utils.js]: can we move this key creation part into a util function?
Coz, someone may miss an space and the keys won't match.

File: pgdjangosql/templates/src/utils/output-section-utils.js [hunk 4]
```diff
@@ -116,18 +169,25 @@ function getFormattedData(
                   flattenedKbdData
                 )
               ) {
-                console.log(
-                  `Formula > ${measure.name} > ${values.name} > ${absKey}`,
-                  formula,
-                  measureValues
-                );
-                values.values[absKey] = eval(formula) + ",1";
+                // console.log(
```
Comment (nabendukarmakar) [inline pgdjangosql/templates/src/utils/output-section-utils.js]: Please keep the console.log

Gunjan and team uses this to debug

File: pgdjangosql/templates/src/utils/app-utils.js (hunk 1 above)
Comment (Sumedh-ravi1604) [reply pgdjangosql/templates/src/utils/app-utils.js]: ok, removed

File: pgdjangosql/templates/src/utils/app-utils.js (hunk 2 above)
Comment (Sumedh-ravi1604) [reply pgdjangosql/templates/src/utils/app-utils.js]: removed

File: pgdjangosql/templates/src/utils/output-section-utils.js (hunk 3 above)
Comment (Sumedh-ravi1604) [reply pgdjangosql/templates/src/utils/output-section-utils.js]: yes, using the getKey() method which is already created

File: pgdjangosql/templates/src/utils/output-section-utils.js (hunk 4 above)
Comment (Sumedh-ravi1604) [reply pgdjangosql/templates/src/utils/output-section-utils.js]: ok

**Pull request closed without merging**